src/goto-programs/write_goto_binary.h

```diff
@@ -12,7 +12,7 @@ Author: CM Wintersteiger
 #ifndef CPROVER_GOTO_PROGRAMS_WRITE_GOTO_BINARY_H
 #define CPROVER_GOTO_PROGRAMS_WRITE_GOTO_BINARY_H
 
-#define GOTO_BINARY_VERSION 5
+#define GOTO_BINARY_VERSION 6
```

Comment:
There is a plan to increment CBMC's major version later number this year, along with adding further potentially breaking changes. There is an assortment of other issues and PRs which have been tagged with "version 6" here - https://github.com/diffblue/cbmc/labels/Version%206

I have no problem with bumping CBMC's major version number if appropriate, but it may be worth discussing the best way to shepherd in all the version 6 changes before proceeding. It might be that we should just mark the regular releases as "pre-release" until these are all in, but I think the major release process should be agreed first.